doc/api/url.md

```diff
@@ -235,6 +235,19 @@ url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
 url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
 ```
 
+Note that `url.resolve()` method depends on the protocol given in the `from` parameter. 
+Some protocols like `https`, `file`, `ftp`, `wss` and `gopher` have special cases.
```

Comment:
The list of protocols that are treated as special cases is very precise and actually defined as this list (see `const slashedProtocol` in `lib/url.js`) :
- `http`
- `https`
- `ftp`
- `gopher`
- `file`

So protocol `wss` is not treated as a special case (as well as `ftps` or any other). I would change the sentence for something like this:

> Some protocols (`http`, `https`, `ftp`, `gopher` and `file`) are treated as special cases. Every other (for instance `ftps` or `wss`) is treated differently whether its URL ends in a slash or not. Although, a uniform behaviour of the method is achieved when the URL ends with a slash.

(note that I moved the last sentence up here)